diff --git a/node_modules/@klarna/react-native-vector-drawable/android/build.gradle b/node_modules/@klarna/react-native-vector-drawable/android/build.gradle
index bc1c1fa..70a657b 100644
--- a/node_modules/@klarna/react-native-vector-drawable/android/build.gradle
+++ b/node_modules/@klarna/react-native-vector-drawable/android/build.gradle
@@ -24,7 +24,9 @@ if (isNewArchitectureEnabled()) {
 android {
     namespace "com.klarna.vectordrawable"
     compileSdkVersion safeExtGet('compileSdkVersion', 31)
-
+    buildFeatures {
+        buildConfig true
+    }
     defaultConfig {
         minSdkVersion safeExtGet('minSdkVersion', 21)
         targetSdkVersion safeExtGet('targetSdkVersion', 31)
