diff --git a/node_modules/@sayem314/react-native-keep-awake/android/build.gradle b/node_modules/@sayem314/react-native-keep-awake/android/build.gradle
index 767229f..0545c37 100644
--- a/node_modules/@sayem314/react-native-keep-awake/android/build.gradle
+++ b/node_modules/@sayem314/react-native-keep-awake/android/build.gradle
@@ -25,7 +25,9 @@ android {
     if (agpVersion.tokenize('.')[0].toInteger() >= 7) {
       namespace "com.sayem.keepawake"
     }
-
+    buildFeatures {
+        buildConfig true
+    }
     compileSdkVersion safeExtGet('compileSdkVersion', 31)
 
     defaultConfig {
