diff --git a/node_modules/react-native-lottie-splash-screen/android/build.gradle b/node_modules/react-native-lottie-splash-screen/android/build.gradle
index ddbf339..def4479 100644
--- a/node_modules/react-native-lottie-splash-screen/android/build.gradle
+++ b/node_modules/react-native-lottie-splash-screen/android/build.gradle
@@ -7,6 +7,7 @@ def DEFAULT_SUPPORT_LIB_VERSION             = "26.1.0"
 def DEFAULT_MIN_SDK_VERSION                 = 16
 
 android {
+    namespace 'org.devio.rn.splashscreen'
     compileSdkVersion rootProject.hasProperty('compileSdkVersion') ? rootProject.compileSdkVersion : DEFAULT_COMPILE_SDK_VERSION
     buildToolsVersion rootProject.hasProperty('buildToolsVersion') ? rootProject.buildToolsVersion : DEFAULT_BUILD_TOOLS_VERSION
 
