diff --git a/node_modules/react-native-version-info/android/build.gradle b/node_modules/react-native-version-info/android/build.gradle
index edcd4fb..13b0720 100644
--- a/node_modules/react-native-version-info/android/build.gradle
+++ b/node_modules/react-native-version-info/android/build.gradle
@@ -5,6 +5,7 @@ def safeExtGet(prop, fallback) {
 apply plugin: 'com.android.library'
 
 android {
+    namespace 'cx.evermeet.versioninfo'
     compileSdkVersion safeExtGet('compileSdkVersion', 27)
     buildToolsVersion safeExtGet('buildToolsVersion', '27.0.3')
 
