diff --git a/node_modules/react-native-document-picker/android/build.gradle b/node_modules/react-native-document-picker/android/build.gradle
index fd24a6e..8b65048 100644
--- a/node_modules/react-native-document-picker/android/build.gradle
+++ b/node_modules/react-native-document-picker/android/build.gradle
@@ -29,7 +29,9 @@ android {
       namespace "com.reactnativedocumentpicker"
     }
     compileSdkVersion getExtOrIntegerDefault('compileSdkVersion')
-
+    buildFeatures {
+        buildConfig true
+    }
     // Used to override the NDK path/version on internal CI or by allowing
     // users to customize the NDK path/version from their root project (e.g. for M1 support)
     if (rootProject.hasProperty("ndkPath")) {
