diff --git a/node_modules/react-native-scoped-storage/android/build.gradle b/node_modules/react-native-scoped-storage/android/build.gradle
index 9a59b70..b2c0601 100644
--- a/node_modules/react-native-scoped-storage/android/build.gradle
+++ b/node_modules/react-native-scoped-storage/android/build.gradle
@@ -19,6 +19,7 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
+  namespace 'com.ammarahmed.scopedstorage'
   compileSdkVersion safeExtGet('compileSdkVersion', 28)
   buildToolsVersion safeExtGet('buildToolsVersion', '28.0.3')
   defaultConfig {
