diff --git a/node_modules/react-native-orientation-locker/android/build.gradle b/node_modules/react-native-orientation-locker/android/build.gradle
index 5f844ae..659e274 100644
--- a/node_modules/react-native-orientation-locker/android/build.gradle
+++ b/node_modules/react-native-orientation-locker/android/build.gradle
@@ -5,6 +5,7 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
+    namespace 'org.wonday.orientation'
     compileSdkVersion safeExtGet('compileSdkVersion', 27)
     buildToolsVersion safeExtGet('buildToolsVersion', '27.0.3')
 
